packages/@aws-cdk/aws-route53-targets/lib/bucket-website-target.ts

```diff
@@ -7,7 +7,7 @@ import {RegionInfo} from '@aws-cdk/region-info';
  * Use a S3 as an alias record target
  */
 export class BucketWebsiteTarget implements route53.IAliasRecordTarget {
-  constructor(private readonly bucket: s3.Bucket) {
+  constructor(private readonly bucket: s3.IBucket) {
```

Comment:
Would this be considered a breaking change, knowing that `Bucket extends BucketBase implements IBucket`?

Reply:
No because IBucket is more general than Bucket.